train/README.md

```diff
@@ -0,0 +1,292 @@
+# LLM Pretraining
+
+## Installation
+
+If you haven't already, make sure to install the requirements:
+
+```bash
+git clone https://github.com/mosaicml/llmfoundry.git
+cd llmfoundry
+pip install -e ".[gpu]"  # or pip install -e . if no NVIDIA GPU
+```
+
+## Dataset preparation
+To run pretraining, you'll need to make yourself a copy of a pretraining dataset. Check out the `llm-foundry/data_prep` folder for detailed instructions.
+
+As a quickstart, here is how to prepare the [C4: Colossal, Cleaned, Common Crawl dataset](https://huggingface.co/datasets/c4).
+We first convert the dataset from its native format (a collection of zipped JSONs)
+to MosaicML's streaming dataset format (a collection of binary `.mds` files).
+Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, etc.)
+and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all ~ just works ~ .
+You can read more about the benefits of using mosaicml-streaming [here](https://streaming.docs.mosaicml.com/en/stable/).
+
+NOTE: If you only want to profile these LLMs, we recommend that you **download and prepare the `train_small` and `val` splits**,
```

Comment:
```suggestion
NOTE: If you only want to profile these LLMs, we recommend that you **download and prepare the `train_small` and `val_small` splits**,
```

Also requires an update [here](https://github.com/mosaicml/llm-foundry/blob/abhi/restruct/data_prep/convert_dataset.py#L39)
(we have val_small, but didn't add it as an option)